lib/assets/javascripts/turbograft/turbohead.coffee

```diff
@@ -14,6 +22,10 @@ class window.TurboHead
       .filter(attributeMatches('nodeName', 'LINK'))
       .filter(noAttributeMatchesIn('href', @activeAssets))
 
+  @reset: ->
```

Comment:
I dislike exposing a test-only method, but the alternative is assigning statics vars to the class.  Either way, we're exposing more context than I want to :/